src/sentry/static/sentry/app/views/discover/queryBuilder.tsx

```diff
@@ -65,15 +66,18 @@ export default function createQueryBuilder(
     query.range = DEFAULT_STATS_PERIOD;
   }
 
-  const defaultProjects = organization.projects.filter(projects => projects.isMember);
+  // TODO(lightweight-org): This needs to be refactored so that queries
+  // do not depend on organization.projects
```

Comment:
What would that look like? Would each query have a project list it was pending on?

Reply:
The caller needs to provide the list of projects, but it looks like only discover/dashboards uses this, so not as big of a deal.